tests/scaleway/scw_utility_test_kubernetes_kapsule_cluster.rs

```diff
@@ -0,0 +1,109 @@
+extern crate test_utilities;
+
+use self::test_utilities::cloudflare::dns_provider_cloudflare;
+use self::test_utilities::utilities::{context, engine_run_test, generate_cluster_id, init, FuncTestsSecrets};
+use tracing::{span, Level};
+
+use qovery_engine::cloud_provider::scaleway::application::Region;
+use qovery_engine::cloud_provider::scaleway::kubernetes::Kapsule;
+use qovery_engine::transaction::TransactionResult;
+
+// Warning: This test shouldn't be ran by CI
+// Note: this test creates the test cluster where all application tests will be ran
+// This is not really a test but a convenient way to create the test cluster if needed to be manually created at some point.
+#[allow(dead_code)]
+#[test]
+#[ignore]
+fn create_scaleway_kubernetes_kapsule_test_cluster() {
```

Comment:
Those tests won't work together because of generated ids. If you looks how it's done AWS side, create and delete are in the same test to avoid this issue

Reply:
Those are not tests, but an handy function to recreate / delete test cluster on the infra, it will allow us to recreate test cluster if needed.